faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Indentifying and Formatting Tabular Dataset 
```

Comment:
```suggestion
title: Identifying and formatting Tabular Datasets
```

Reply:
@Beachyesh this commit was not done completely, do you know that you can do it directly from this comment here by clicking on "Commit suggestion" instead of editing the file?